app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -591,7 +591,7 @@ class BrowserFragment : LocaleAwareFragment(), BrowserScreen, LifecycleOwner, Ba
 
     override fun onConfigurationChanged(newConfig: Configuration) {
         super.onConfigurationChanged(newConfig)
-        updateBottomBarHeight()
+        updateBottomBarLayout()
```

Comment:
is there any way the custom view can be aware of the orientation changing event?